packages/valory/skills/dynamic_nft_abci/behaviours.py

```diff
@@ -102,9 +102,11 @@ def get_token_id_to_address(
     ) -> Generator[None, None, Tuple[dict, Optional[int]]]:
         """Get token id to address data."""
         try:
-            from_block = self.synchronized_data.ceramic_db["module_data"][
-                "dynamic_nft"
-            ]["last_parsed_block"]
+            from_block = int(
+                self.synchronized_data.ceramic_db["module_data"]["dynamic_nft"][
+                    "last_parsed_block"
+                ]
+            )
```

Comment:
So if a dev in the future removes the `int()` part no tests will fail? We need to update the unit tests for this.

Reply:
It's because this variable is sent to the contract, which is mocked anyway so it does not matter what type it is. I tried checking its type in the mock_contract_api_request but this mehod does not support checking custom parameters. In any case, I'm aware of this problems and I opened a ticket yesterday to do data validation, so this code wont be necessary.

Reply:
Please link the issue that you mentioned and we can merge :)

Reply:
[Here's the ticket](https://trello.com/c/usSfKU8p/2090-feature-data-validation-on-ceramic-loading-writing).